packages/apps/src/Endpoints/index.tsx

```diff
@@ -270,7 +287,7 @@ function Endpoints ({ className = '', offset, onClose }: Props): React.ReactElem
                 : <Button
                   className='customButton'
                   icon='save'
-                  isDisabled={!isUrlValid || isKnownUrl}
+                  isDisabled={!isUrlValid || isKnownUrl || getApiType(apiUrl).type === 'json-rpc'}
```

Comment:
This goes with the above - so either hide completely (don't fallback to previous) or show the previous valid.